src/program/apis/listrr_api.py

```
@@ -55,13 +55,13 @@ def get_items_from_Listrr(self, content_type, content_lists) -> list[MediaItem]
                     url = f"api/List/{content_type}/{list_id}/ReleaseDate/Descending/{page}"
                     response = self.request_handler.execute(HttpMethod.GET, url)
                     data = response.data
-                    total_pages = data.get("pages", 1)
-                    for item in data.get("items", []):
-                        imdb_id = item.get("imDbId")
+                    total_pages = getattr(data, "pages", 1)
+                    for item in getattr(data, "items", []):
+                        imdb_id = getattr(item,"imDbId")
```

Ruff (0.8.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)